support-frontend/assets/pages/paper-subscription-landing/paperSubscriptionLandingPage.tsx

```diff
@@ -61,11 +67,7 @@ const PaperLandingPage = ({
 			product: 'Paper',
 			componentType: 'ACQUISITIONS_BUTTON',
 		})();
-		window.history.replaceState(
-			{},
-			null,
-			paperSubsUrl(newTab === HomeDelivery),
-		);
+		window.history.replaceState({}, '', paperSubsUrl(newTab === HomeDelivery));
```

Comment:
the middle param is unused (it's literally called that in the type signature) but it should be passed an empty string https://developer.mozilla.org/en-US/docs/Web/API/History/replaceState